src/widget/form/settings/generalform.cpp

```diff
@@ -84,8 +84,6 @@ static QStringList langs = {"Български",
                             "简体中文"};
 
 static QStringList timeFormats = {"hh:mm AP", "hh:mm", "hh:mm:ss AP", "hh:mm:ss"};
```

Comment:
No. According to the `QTime` documentation, appending `AP` activates AM/PM display, which results in 0-12 hour display. Thus `HH` is an invalid format string.